app/models/work.rb

```diff
@@ -0,0 +1,20 @@
+class Work < ApplicationRecord
+  # https://guides.rubyonrails.org/association_basics.html #Lee taught me this!!!
+  has_many :votes, dependent: :destroy
+  has_many :users, through: :votes
+
+  validates :title, presence: true
+  validates :title, uniqueness: {scope: :category}
+
+  def self.top_ten(category)
+    return Work.where(category: category).max_by(10) {|work| work.votes.count}
+  end
+
+  def self.max_votes
+    return Work.all.max_by {|work| work.votes.count}
+  end
+
+  def self.category_sort(category)
+    return Work.where(category: category).sort_by{|work| -work.votes.count}
```

Comment:
It might be clearer to use `reverse` here instead of sorting by a negative:

```suggestion
    return Work.where(category: category).sort_by{|work| work.votes.count}.reverse
```